sympy/series/limits.py

```diff
@@ -128,6 +128,47 @@ def heuristics(e, z, z0, dir):
     return rv
 
 
+def return_piecewise(expr, z0, cdir):
+    """Helper function to evaluate limits when the point, ``z0``, is passed as a Symbol.
+    Returns the limit in piecewise form over the complete real number line from
+    Negative Infinity to Positive Infinity.
+
+    Note:This method is currently implemented for expressions having finite discontinuities.
+
+    Examples
+    ========
+
+    >>> from sympy import limit, sin, log
```

Comment:
It probably makes sense to actually import and use the new function here and/or add an example in `limit` showing that it can handle symbolic points.

Reply:
Yeah makes sense that we should be adding couple of exams in limit docstring too !